src/reframe.js

```diff
@@ -18,8 +18,8 @@ export default function reframe(target, cName) {
     if (hasClass) return;
 
     // general targeted <element> sizes
-    const h = frame.offsetHeight;
-    const w = frame.offsetWidth;
+    const h = parseInt(frame.getAttribute('height'));
```

Comment:
I think a ternary might solve both, the issue you're having and your other note.

**In example:**
```javascript 
const h = frame.getAttribute('height') !== 'undefined' ? 
  parseInt(frame.getAttribute('height')) : 
  frame.offsetHeight
```
^ I'm not 100% that that is correct but what about something like that? 😁